tenants/ai-example/lmeval-lab/base/serving-runtime.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: ServingRuntime
+metadata:
+  annotations:
+    opendatahub.io/accelerator-name: ""
+    opendatahub.io/apiProtocol: REST
+    opendatahub.io/recommended-accelerators: '["nvidia.com/gpu"]'
+    opendatahub.io/template-display-name: Qwen2.5-0.5B-Instruct for TrustyAI LMEval lab
+    opendatahub.io/template-name: qwen-runtime
+    openshift.io/display-name: Qwen2.5-0.5B-Instruct for TrustyAI LMEval lab
+  labels:
+    opendatahub.io/dashboard: "true"
+  name: qwen-instruct
+spec:
+  annotations:
+    prometheus.io/path: /metrics
+    prometheus.io/port: "8080"
+    openshift.io/display-name: Qwen2.5-0.5B-Instruct for TrustyAI LMEval lab
+  labels:
+    opendatahub.io/dashboard: "true"
+  containers:
+  - name: kserve-container
+    image: quay.io/opendatahub/vllm:stable-849f0f5
```

Comment:
We should avoid using opendatahub images unless we have to.

Do you know what version of vllm this is?

This is the officially supported vllm version 0.7.3 that is shipping with 2.18:
`quay.io/modh/vllm@sha256:4f1f6b5738b311332b2bc786ea71259872e570081807592d97b4bd4cb65c4be1`

Reply:
Done.
Image quay.io/modh/vllm@sha256:4f1f6b5738b311332b2bc786ea71259872e570081807592d97b4bd4cb65c4be1 is used and validated via LM-Eval run.